cmd/root.go

```diff
@@ -128,10 +104,20 @@ func executePhases(cmd cobra.Command, progress *phases.Progress) error {
 	}
 
 	if cmd.Flags().Changed(cmdFlagKeyBitriseYML) {
-		progress.BitriseYML = &cmdFlagBitriseYML
+		DSL, found, err := phases.OpenAndParseDSLFile(cmdFlagKeyBitriseYML)
```

Comment:
Long function names describing what they do with "And" effectively encode implementation details in their name. This makes the code harder to reason about, because you have to keep a long and detailed name in your mental model.

Functions can be named based on the expected outcome. In this case I'd recommend simply `ParseDSLFile`.

Reply:
Just removed the "open" logic from this function.